doom/src/main/java/net/nullsum/freedoom/EntryActivity.java

```diff
@@ -1,49 +1,45 @@
 package net.nullsum.freedoom;
 
+import android.Manifest;
 import android.app.ActionBar;
 import android.app.ActionBar.Tab;
 import android.app.Fragment;
 import android.app.FragmentTransaction;
 import android.content.Context;
+import android.content.pm.PackageManager;
 import android.content.res.Resources;
 import android.os.Bundle;
-import android.support.annotation.StringRes;
+import android.support.v4.app.ActivityCompat;
 import android.support.v4.app.FragmentActivity;
 import android.support.v4.content.ContextCompat;
-import android.support.v4.app.ActivityCompat;
 import android.util.Log;
 import android.view.KeyEvent;
 import android.view.Menu;
 import android.view.MotionEvent;
-import android.content.pm.PackageManager;
 import android.widget.Toast;
 
-import static android.Manifest.*;
-
 import com.beloko.touchcontrols.GamePadFragment;
 
 import java.util.Arrays;
 
 
-public class EntryActivity extends FragmentActivity  {
-
-    GamePadFragment gamePadFrag;
-    String LOG = "EntryActivity";
+public class EntryActivity extends FragmentActivity {
 
     /**
      * The serialization (saved instance state) Bundle key representing the
      * current tab position.
      */
     private static final String STATE_SELECTED_NAVIGATION_ITEM = "selected_navigation_item";
     private static final int PERMISSIONS_REQUEST_WRITE_EXTERNAL_STORAGE = 1;
-    private Resources res;
+    GamePadFragment gamePadFrag;
+    String LOG = "EntryActivity";
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
-        if (ContextCompat.checkSelfPermission(this, permission.WRITE_EXTERNAL_STORAGE) != PackageManager.PERMISSION_GRANTED) {
-            ActivityCompat.requestPermissions(this, new String[]{ permission.WRITE_EXTERNAL_STORAGE }, PERMISSIONS_REQUEST_WRITE_EXTERNAL_STORAGE);
+        if (ContextCompat.checkSelfPermission(this, Manifest.permission.WRITE_EXTERNAL_STORAGE) != PackageManager.PERMISSION_GRANTED) {
```

Comment:
What does this do?

Reply:
Just uses an additional qualifier (I think it's called) instead of needing to have a static import. Doesn't change anything really